src/Nest/Ingest/Processors/DissectProcessor.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	/// <summary>
+	/// Similar to the Grok Processor, dissect also extracts structured fields out of a single text field
+	/// within a document. However unlike the Grok Processor, dissect does not use Regular Expressions.
+	/// This allows dissect’s syntax to be simple and for some cases faster than the Grok Processor.
```

Comment:
This allows dissect’s syntax to be simple and, for some cases, faster than the Grok Processor.